### PR TITLE
Bugfix: removed .as_list() call from an object that is already a list…

### PR DESCRIPTION
**Pull request by babua**
…. This was causing a crash with non-zero delta weights in SpectralLoss

### CODE REVIEW

**Comment by babua** [comment]
@googlebot I signed it!

**Comment by babua** [comment]
Added a test. I was thinking if this should also be tested in `losses_test` by using `SpectralLoss` also, but I guess it's out of scope of this commit.

File: ddsp/spectral_ops_test.py [hunk 1]
```diff
@@ -39,6 +39,28 @@ def test_tf_and_np_are_consistent(self):
     # TODO(jesseengel): The phase comes out a little different, figure out why.
     self.assertAllClose(np.abs(s_np), np.abs(s_tf), rtol=1e-3, atol=1e-3)
 
+  def test_diff(self):
```
Comment (jesseengel) [inline ddsp/spectral_ops_test.py:42]:
```suggestion
  def test_diff_shape_is_correct(self):
```

**Comment by jesseengel** [comment]
Nice! Seems like you just need to fix up to fit the style guidelines: https://travis-ci.org/github/magenta/ddsp/builds/678463894

**Comment by jesseengel** [comment]
Thanks for the contribution! Sorry to swoop in, but I incorporated your changes into a new PR #87 that fixes the problems in #86, so I'm going to close this one for now as it's been in waiting for a few days. Thank you again for pointing this out and providing a fix!

**Pull request closed without merging**